pkg/venafi/tpp/connector.go

```diff
@@ -403,6 +403,33 @@ func prepareRequest(req *certificate.Request, zone string) (tppReq certificateRe
 	tppReq.CASpecificAttributes = append(tppReq.CASpecificAttributes, nameValuePair{Name: "Origin", Value: origin})
 	tppReq.Origin = origin
 
+	if req.ValidityHours > 0 {
+
+		expirationDateAttribute := ""
+
+		switch req.IssuerHint {
+		case "MICROSOFT":
```

Comment:
If we define constants for the issuer names, we don't need to create new strings each time we need to use them

Reply:
fixed on most recent commit